src/sentry/static/sentry/less/includes/select2.less

```diff
@@ -405,9 +405,9 @@ disabled look for disabled choices in the results dropdown
   display: none;
 }
 
-.select2-more-results.select2-active {
-  background: #f4f4f4 url('select2-spinner.gif') no-repeat 100%;
-}
+// .select2-more-results.select2-active {
+//   background: #f4f4f4 url('~select2/select2-spinner.gif') no-repeat 100%;
+// }
```

Comment:
Commented out because there doesn't seem to be a way to configure `file-loader` to look up assets inside of `node_modules`. Going to file a ticket.

I've verified with @ckj that these rules aren't being used.